docs/source/installation.rst

```diff
@@ -2,152 +2,133 @@ Installation
-working under a virtual environment, which could be set as follows:
+            It is best to attempt ``venv`` based installations and follow up with ComputeCanada technical support as MANY specially compiled packages (e.g. numpy) are exclusively available for Compute Canada HPC environment.
 
-::
+            If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules as `mentioned here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_.
 
-    virtualenv venv-ivadomed
-    source venv-ivadomed/bin/activate
 
+Step 2: Install ``ivadomed``
+++++++++++++++++++++++++++++
 
-.. warning::
-   If the default Python version installed in your system does not fit the version requirements, you might need to specify a version of Python associated with your virtual environment:
 
-   ::
+    .. tabs::
 
-     virtualenv venv-ivadomed --python=python3.6
+        .. tab:: PyPI Installation
 
-Step 2: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+            Install ``ivadomed`` and its requirements from
+            `PyPI <https://pypi.org/project/ivadomed/>`__:
 
+            ::
 
-::
+                pip install --upgrade pip
 
-    git clone https://github.com/ivadomed/ivadomed.git
-    cd ivadomed
- 
+                pip install ivadomed
 
-Step 3: Install PyTorch 1.5 and TorchVision
-+++++++++++++++++++++++++++++++++++++++++++
+        .. tab:: Repo Installation (Advanced or Developer)
 
-If you have a compatible NVIDIA GPU that supports CUDA, run the following command:
+            Bleeding-edge developments are available on the project's master branch
+            on Github. Install ``ivadomed`` from source:
 
-::
-   
-   pip install -r requirements_gpu.txt
+            ::
 
+                git clone https://github.com/ivadomed/ivadomed.git
 
-If you do not have a compatible GPU, run the following installer to use ``ivadomed`` with CPU.
+                cd ivadomed
 
+                pip install -e .
 
-::
 
-    pip install -r requirements.txt
+Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+    .. tabs::
 
-Step 4: Install from release (recommended)
-++++++++++++++++++++++++++++++++++++++++++
+        .. tab:: CPU Support
 
-Install ``ivadomed`` and its requirements from
-`Pypi <https://pypi.org/project/ivadomed/>`__:
+            If you plan to run ``ivadomed`` on CPU only, install pytorch per instructions `provided here for 1.8 <https://pytorch.org/get-started/previous-versions/#wheel>`_.
```

Comment:
I find this line a bit confusing.

If I understand correctly:
- if installed with PyPI, torch must be installed from the external link
- if the repo is cloned, it can be installed using requirements

If that is the case, it would need to be more explicitly explained.
Also, linking to external site for instructions can be confusing for user, so I would suggest listing the command to run directly in our instructions.

Reply:
Same thought! (https://github.com/ivadomed/ivadomed/pull/969#discussion_r748739159) :smile:

Reply:
Good point! Adopted in conjunction with Joshua's comment. Explicitly stated now for all scenarios.